### PR TITLE
Corrected the folder of METplus log files for WAFS

### PR DESCRIPTION
**Pull request by YaliMao-NOAA**
Corrected the folder of METplus log files to be attached to the general log file

## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

File: scripts/stats/wafs/exevs_wafs_atmos_stats.sh [hunk 1]
```diff
@@ -8,9 +8,7 @@
 set -x
 
 cd $DATA
-if [ -s wafs_stat.cmdfile ];then
-	rm wafs_stat.cmdfile
-fi
+rm -f wafs_stat.cmdfile
```
Comment (PerryShafran-NOAA) [inline scripts/stats/wafs/exevs_wafs_atmos_stats.sh:12]: @YaliMao-NOAA Why are we removing the if block here?  If the wafs_stat.cmdfile doesn't exist, then line 11 here will cause a "No such file" warning,  wouldn't it?

**Comment by PerryShafran-NOAA** [comment]
@YaliMao-NOAA I have begun work on testing this PR.

File: scripts/stats/wafs/exevs_wafs_atmos_stats.sh (hunk 1 above)
Comment (PerryShafran-NOAA) [reply scripts/stats/wafs/exevs_wafs_atmos_stats.sh:12]: Oh, the -f prevents that No such file message from happening.  I stand corrected.

**Comment by PerryShafran-NOAA** [comment]
@YaliMao-NOAA The test is finished.  

See the .o file here:  /lfs/h2/emc/vpppg/noscrub/perry.shafran/pr418test/EVS/dev/drivers/scripts/stats/wafs/jevs_wafs_stats.o113818180

See the output here: /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/stats/wafs/wafs.20240123

**Comment by YaliMao-NOAA** [comment]
@PerryShafran-NOAA Both of the log file and the stat output look good to me

**Comment by PerryShafran-NOAA** [comment]
Great!  I will check the code and merge.

**Review by PerryShafran-NOAA — APPROVED** [review]

**Pull request merged**